sdk/storage/azdatalake/file/client_test.go

```diff
@@ -5151,6 +5151,48 @@ func (s *RecordedTestSuite) TestFileGetPropertiesWithCPK() {
 	_require.Equal(testcommon.TestCPKByValue.EncryptionKeySHA256, response.EncryptionKeySHA256)
 }
 
+func (s *RecordedTestSuite) TestFileGetPropertiesWithEncryptionContext() {
```

Comment:
Please also add a test for `DownloadStream` where encryption context is returned in the response.